app/views/simplified-account/settings/email-notifications/templates.njk

```diff
@@ -0,0 +1,181 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Email notifications - {{ currentService.name }} {{ currentGatewayAccount.full_type }}
+{% endblock %}
+
+{% block settingsContent %}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: backLink
+  }) }}
+
+  {% if not isAdminUser %}
+    {{ govukInsetText({
+      text: "You don’t have permission to manage settings. Contact your service admin if you would like to manage 3D Secure,
+        accepted card types, email notifications, billing address or mask card numbers or security codes for MOTO services.",
+      classes: "service-settings-inset-text--grey"
+    }) }}
+  {% endif %}
+
+  <h1 class="govuk-heading-l">Email templates</h1>
+
+  <p class="govuk-body-l">
+    GOV.UK Pay can send notifications to users.
+  </p>
+
+  {% set confirmationEmailHtml %}
+    <h2 class="govuk-heading-l">Confirmation email</h2>
+    {{ govukTable({
+      rows: [
+        [
+          {
+            text: 'From'
+          },
+          {
+            text: 'GOV.UK Pay'
+          }
+        ],
+        [
+          {
+            text: 'To'
+          },
+          {
+            text: '[email address]'
+          }
+        ],
+        [
+          {
+            text: 'Subject'
+          },
+          {
+            text: 'Your payment of £[amount] to ' + serviceName + ' was successful'
+          }
+        ]
+      ]
+    }) }}
+
+    <div class="pay-email-template-body">
+      <h3 class="govuk-heading-m pay-email-heading">Your payment of £<span class="govuk-grey-text">[amount]</span> to <span class="govuk-grey-text">{{ serviceName }}</span> was successful</h3>
+      <ul class="govuk-list">
+        <li>Payment for: <span class="govuk-grey-text">[description]</span></li>
+        <li>Reference: <span class="govuk-grey-text">[service reference]</span></li>
+        <li>Date of payment: <span class="govuk-grey-text">[date]</span></li>
+      </ul>
+      {% if customEmailText %}
+        <p class="govuk-body qa-custom-p">
+          {{ customEmailText | striptags(true) | escape | nl2br }}
+        </p>
+      {% else %}
+        <span class="govuk-body govuk-grey-text qa-custom-p">*Optional custom paragraph - add this below*</span>
+      {% endif %}
+      <p class="govuk-body">This email address is not monitored. If you have any questions about your payment, contact the service you made the payment to directly.</p>
```

Comment:
according to the figma, there needs to be a space between the 'optional custom paragraph' and the 'This email is not monitored..'